src/tabs/docs/readme.md

```diff
@@ -26,9 +26,10 @@ AngularJS version of the tabs directive.
  	_(Defaults: false)_ :
  	Whether tab is currently selected.
 
- * `disabled` <i class="glyphicon glyphicon-eye-open"></i>
+ * `disable` <i class="glyphicon glyphicon-eye-open"></i>
  	_(Defaults: false)_ :
  	Whether tab is clickable and can be activated.
+ 	Note that this deprecates the "disabled" attribute  
```

Comment:
Or
<pre>
Note that this was previously the `disabled` attribute, which is now deprecated.
</pre>